astropy/units/equivalencies.py

```diff
@@ -116,13 +116,20 @@ def spectral_density(wav, factor=None):
     h_cgs = _si.h.cgs.value  # erg * s
     hc = c_Aps * h_cgs
 
+    # flux density
     fla = cgs.erg / si.angstrom / si.cm ** 2 / si.s
     fnu = cgs.erg / si.Hz / si.cm ** 2 / si.s
     nufnu = cgs.erg / si.cm ** 2 / si.s
     lafla = nufnu
     photlam = astrophys.photon / (si.cm ** 2 * si.s * si.AA)
     photnu = astrophys.photon / (si.cm ** 2 * si.s * si.Hz)
 
+    # luminosity density
+    lnu = cgs.erg / si.s / si.Hz
+    lla = cgs.erg / si.s / si.angstrom
+    nulnu = cgs.erg / si.s
+    lalla = nulnu
```

Comment:
While we're at it, perhaps also add photon luminosities? I.e., photon/s/Hz and photon/s/A